### PR TITLE
Dion Dev Onboarding First PR

### PR DESCRIPTION
**Pull request by dionyichia**
[//]: # BeepBoopBeepBoop

### Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

- Completed Assignment 1, made app look like figma
- Made it scrollable, added repeat post to show functionality
- Tried using next image component but ran into some issues passing url as src so defaulted to img instead

### Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy! Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."

image

## How to review
[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'

Nil 

## Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."

Nil 

### Online sources
[//]: # 'Copy links to any tutorials or documentation that was useful to you when working on this PR'

Nil

### Related PRs
[//]: # "Add related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

Nil

CC: @kylezryr

### CODE REVIEW

**Comment by dionyichia** [comment]
I fixed the eslint errors and switched to using Next Image for all imgs and recommited

File: app/components/Header.tsx
Comment (kylezryr) [inline app/components/Header.tsx:1]: wait you weren't even supposed to implement this LOL but great job man looks great

File: app/components/UserProfile.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+export interface UserProfileProps {
+  username: string;
+  pic?: string;
+}
+
+const UserProfile = (props: UserProfileProps) => {
+  return (
+    <div className="user-profile">
+      <div>
```
Comment (kylezryr) [inline app/components/UserProfile.tsx]: very small nitpick: the profile icon and the name aren't center aligned as you can see in the image, i think you need to center the ProfileIcon component itself as well

image

**Review by kylezryr — APPROVED** [review]
bro this looks great omg great job! you made the components and everything which was acc supposed to be this weeks assignment LMAO, so really great man !

**Comment by kylezryr** [comment]
you can try merging this yourself! make sure to select the "squash and merge" option so that all the commits get combined into 1 commit, keeping our code clean!

File: app/components/UserProfile.tsx (hunk 1 above)
Comment (dionyichia) [reply app/components/UserProfile.tsx]: Alrighty, I hope i understood you correctly, I've added styling to the profile pic container so that it align-centers with username. Sorrys for the oversight. 


image
